docs/public/_redirects

```diff
@@ -93,6 +93,7 @@ https://v3-9-0.mui.com/* https://v3.mui.com/:splat 301!
 /customization/components/ /customization/how-to-customize/ 301
 /customization/globals/ /customization/theme-components/ 301
 # 2021
+/api/root-ref/ https://v4.mui.com/api/root-ref/ 301
```

Comment:
Backlinks pointing to this page

<img width="734" alt="Screenshot 2021-12-23 at 00 02 38" src="https://user-images.githubusercontent.com/3165635/147164587-2af10345-eafe-46e9-b898-7c3bfdfe31af.png">